app/config/routing.yml

```diff
@@ -1,6 +1,9 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
+fos_oauth_server_token:
```

Comment:
Shouldn't we add this to an ApiBundle instead of adding it to main routing file? And also add some simple test of authentication to ensure it works?